client/dist/esm/streams.js

```diff
@@ -50,7 +50,17 @@ export function decodeContent() {
         transform(value, controller) {
             // Base64 decode the `content` field and send it as a chunk.
             if (value.content?.length) {
-                controller.enqueue(atob(value.content));
+                // The `atob` function does not work properly if the decoded content contains any byte
```

Comment:
I know we are not running in `node` but in the UI we have stopped using `atob` and `btoa` and replaced with 'Buffer` and `.toString('base64')`

https://dev.to/2ezpz2plzme/btoa-replacement-in-nodejs-3k6g

Reply:
I don't mind trying this out on my local and running the tests

Reply:
Never mind - as that will be a pain polyfilling Node stuff.